ocean_lib/models/v4/erc20_enterprise.py

```diff
@@ -25,3 +26,20 @@ def buy_from_dispenser_and_order(
         return self.send_transaction(
             "buyFromDispenserAndOrder", (order_params, dispenser_address), from_wallet
         )
+
+    def create_dispenser(
```

Comment:
please delete this function, it is inherited from ERC20.

Reply:
Unfortunately, I cannot use the `create_dispenser` function from the ERC20. The parameters for the `create_dispenser` function in enterprise ERC20 are different than the ERC20.